code/game/objects/items/stacks/stack.dm

```diff
@@ -195,15 +194,18 @@
 			interact(usr)
 			return
 
-/obj/item/stack/proc/use(var/used)
+/obj/item/stack/proc/use(used)
 	if(amount < used)
 		return 0
 	amount -= used
 	if(amount < 1) // Just in case a stack's amount ends up fractional somehow
+		if(isrobot(loc))
+			var/mob/living/silicon/robot/R = loc  //Horrifying cyborg snowflake code that allows stacks to GC and cyborgs not to horrendously break
+			if(locate(src) in R.module.modules)
+				R.module.modules -= src
 		if(usr)
-			usr.unEquip(src, 1)
-		spawn()
```

Comment:
Why was this here.

Reply:
If I had to guess, it's because stack code is from before qdel, so anything calling use() could experience a sudden long ass pause for del to work it's thing

Reply:
That'd make sense.